### PR TITLE
Add netstandard1.6 support

### PR DESCRIPTION
**Pull request by shep1987**
As microsoft have now released: Microsoft.ApplicationInsights.AspNetCore

I wanted to loose the "dotnet5.4" dependancy so i could use this package in our cross platform project.

Here is the work i had to do to achieve this

### CODE REVIEW

File: src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs [hunk 1]
```diff
@@ -114,7 +114,6 @@ public static class LogEventExtensions
             var exceptionTelemetry = new ExceptionTelemetry(logEvent.Exception)
             {
                 SeverityLevel = logEvent.Level.ToSeverityLevel(),
-                HandledAt = ExceptionHandledAt.UserCode,
```
Comment (nblumhardt) [inline src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs]: If this is unavailable in the .NET Core version, could we `#if` around it?

File: src/Serilog.Sinks.ApplicationInsights/project.json [hunk 2]
```diff
@@ -9,18 +9,29 @@
...
         "dotnet5.4"
       ]
+    },
+    "netstandard1.6": {
+        "dependencies": {
+          "Microsoft.ApplicationInsights.AspNetCore": "2.0.0"
+        }
+      }
+    },
+    "buildOptions": {
```
Comment (nblumhardt) [inline src/Serilog.Sinks.ApplicationInsights/project.json]: The indentation is mixed up here.

**Comment by nblumhardt** [comment]
Thanks for the PR! Just a couple of brief comments. Cheers!

File: src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs (hunk 1 above)
Comment (shep1987) [inline src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs]: I will #if around it but i believe the data is no longer saved into app insights you must log it as a custom property, understand wanting to keep the changes to a minimum though!

https://docs.microsoft.com/en-us/dotnet/api/microsoft.applicationinsights.datacontracts.exceptionhandledat

File: src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs [hunk 3]
```diff
@@ -110,11 +110,13 @@ public static class LogEventExtensions
         {
             if (logEvent == null) throw new ArgumentNullException("logEvent");
             if (logEvent.Exception == null) throw new ArgumentException("Must have an Exception", "logEvent");
-            
+
             var exceptionTelemetry = new ExceptionTelemetry(logEvent.Exception)
             {
                 SeverityLevel = logEvent.Level.ToSeverityLevel(),
+#if !NETSTANDARD1_6
```
Comment (nblumhardt) [inline src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs]: We've found the target framework monikers are a bit fragile to use in `#if` blocks; would it be possible to use:

```csharp
#if EXCEPTION_TELEMETRY_HANDLED_AT
...
```

and then `"buildOptions": { "define": ["EXCEPTION_TELEMETRY_HANDLED_AT"] }` in the two targets that support it?

Comment (shep1987) [inline src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs]: Sure thing, out of interest is this just due to the possibity of breaking things if new frameworks are added?

Comment (nblumhardt) [inline src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs]: Thanks - yes, as more targets are added, or perhaps when one target is widened, e.g. if AI started supporting .NET Standard 1.5 and we changed the project.json file to accommodate it, these can be a pain to keep updated.

**Comment by nblumhardt** [comment]
Thanks for the changes. LGTM. @jbattermann ?

**Comment by jbattermann** [comment]
LGTM 👍  Thanks @shep1987 & @nblumhardt !

**Pull request merged**